test/UnsafeTest/src_90/org/openj9/test/unsafe/TestUnsafeAccess.java

```diff
@@ -24,6 +24,7 @@
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 import org.testng.log4testng.Logger;
+import org.testng.AssertJUnit;
```

Comment:
Please use org.testng.Assert as AssertJUnit is mostly used for converted junit methods.

Reply:
updated.